scout/utils/scout_requests.py

```diff
@@ -319,6 +318,26 @@ def fetch_mim_files(api_key, mim2genes=False, mimtitles=False, morbidmap=False,
     return mim_files
 
 
+def fetch_orpha_files(product4=False, product6=False):
+    """Fetch the necessary files
+    https://www.orphadata.com/data/xml/en_product{nr}.xml
+    Returns:
+        orpha_files(dict): A dictionary with selected files from orphadata
+    """
+    LOG.info("Fetching orpha files from orphadata")
+
+    orphadata_en_product4_url = "https://www.orphadata.com/data/xml/en_product4.xml"
```

Comment:
Rename this variable to something that makes sense to us and move it to scout.constants.phenotypes, so you can import it here and also in the test. Better to specify it in one place only so if it changes you fix it there and you don't have to go and chase it down to all the places where it is written.